molgenis-data-migrate/src/main/java/org/molgenis/data/migrate/bootstrap/MolgenisUpgradeBootstrapper.java

```diff
@@ -32,7 +31,6 @@ public void bootstrap() {
     upgradeService.addUpgrade(new Step36EnableDataRowEdit(dataSource));
     upgradeService.addUpgrade(new Step37AddSettingsPluginToMenu(dataSource));
     upgradeService.addUpgrade(new Step38AddAnonymouseRoleToDb(dataSource));
-    upgradeService.addUpgrade(new Step39CreateRootPackageGroups(dataSource));
```

Comment:
Why? According to http://wiki.gcc.rug.nl/ticket/6336 we ```Wrote a migration step that creates groups for packages-without-parent ```. I don't see how this works after your changes.